integration-tests/test-suite/partials/layout.xml

```diff
@@ -474,5 +474,38 @@
       <url href="comparators/layout/failed.jsp"/>
     </urls>
   </test>
+
+	<test name="F-comparator-Layout-dynamic-element-at-bottom-of-long-page">
+		<collect>
+			<open/>
+			<resolution width="767"/>
+			<sleep duration="2000"/>
+			<resolution width="767"/>
+			<screen css=".dynamic2"/>
+		</collect>
+		<compare>
+			<screen comparator="layout"/>
+		</compare>
+		<urls>
+			<url href="comparators/layout/long_page.jsp"/>
+		</urls>
+	</test>
+
+	<test name="S-comparator-Layout-long-page-without-dynamic-at-bottom">
+		<collect>
+			<open/>
+			<resolution width="767"/>
```

Comment:
Could you please leave here `ToDo` comment with info that resolution-sleep-resolution workaround should be removed after #357 is fixed?

Reply:
Done